src/plugins/gui.default/template.html

```diff
@@ -61,4 +63,24 @@
     </ul>
   {{/if}}
 </li>
-</script>
\ No newline at end of file
+</script>
+
+
+<script id="pageTmpl" type="text/x-template">
+  profile
+  {^{menu ~userProfile width=150}}
+    <li>
+      <div>
+        <span class="ui-icon ui-icon-user"></span>
+        {{:name}}
+      </div>
+    </li>
+    <li>
+      <div>
+        <span class="ui-icon ui-icon-close" data-i18n="account-close"></span>Salir
```

Comment:
Suggest to use account.logout instead of account-close for the language string key.